be/src/bench/parquet_writer_bench.cpp

```diff
@@ -0,0 +1,236 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <benchmark/benchmark.h>
+#include <glog/logging.h>
+#include <gtest/gtest.h>
+
+#include <memory>
+#include <random>
+
+#include "bench.h"
+#include "column/chunk.h"
+#include "common/statusor.h"
+#include "exprs/runtime_filter.h"
+#include "exprs/runtime_filter_bank.h"
+#include "formats/parquet/file_writer.h"
+#include "formats/parquet/parquet_test_util/util.h"
+#include "fs/fs.h"
+#include "fs/fs_posix.h"
+#include "runtime/descriptor_helper.h"
+#include "simd/simd.h"
+#include "testutil/assert.h"
+#include "util/time.h"
+
+namespace starrocks {
+namespace parquet {
+namespace {
+
+// TODO: chunk size of 4096
+inline std::shared_ptr<Chunk> make_chunk(int num_rows) {
+    auto chunk = std::make_shared<Chunk>();
+    auto col = Bench::create_series_column(TypeDescriptor::from_logical_type(TYPE_INT), num_rows);
+    chunk->append_column(col, 0);
+    return chunk;
+}
+
+inline std::shared_ptr<arrow::Table> make_arrow_table(int num_rows) {
+    std::vector<int32_t> values(num_rows);
+    std::iota(values.begin(), values.end(), 0);
+    arrow::Int32Builder i32builder;
+    PARQUET_THROW_NOT_OK(i32builder.AppendValues(values));
+    std::shared_ptr<arrow::Array> i32array;
+    PARQUET_THROW_NOT_OK(i32builder.Finish(&i32array));
+
+    std::shared_ptr<arrow::Schema> schema = arrow::schema({arrow::field("int", arrow::int32())});
+
+    return arrow::Table::Make(schema, {i32array});
+}
+
+inline std::shared_ptr<::parquet::WriterProperties> make_property() {
+    //    ::parquet::WriterProperties::Builder builder;
```

Comment:
why not make it as configurable?

Reply:
For now, keeping two writers' configurations same is enough